pkg/apis/velero/v1/labels_annotations.go

```diff
@@ -75,4 +75,7 @@ const (
 	// ResourceTimeoutAnnotation is the annotation key used to carry the global resource
 	// timeout value for backup to plugins.
 	ResourceTimeoutAnnotation = "velero.io/resource-timeout"
+
+	// AsyncOperationIDLabel is the label key used to identify the async operation ID
+	AsyncOperationIDLabel = "velero.io/async-operation-id"
```

Comment:
Are we using this anywhere?

Reply:
This is used by the CSI plugin. I thought the Velero server also needs it.
If not, just keeping it in CSI plugin code repository is good enough.